src/prompts/questions.js

```diff
@@ -35,6 +36,13 @@ function authorQuestions({name, email, url}) {
 export async function prompt({npmAccount, author}, ciServices, hosts, visibility, vcs, decisions) {
   const npmConf = npmConfFactory();
 
+  let maybeLoggedInNpmUsername;
+  try {
+    maybeLoggedInNpmUsername = (await execa('npm', ['whoami'])).stdout;
+  } catch (failedExecutionResult) {
+    warn('No logged in user found with `npm whoami`. Login with `npm login` to enable default project scoping.');
```

Comment:
```suggestion
    warn('No logged in user found with `npm whoami`. Login with `npm login` to use your npm account name as the package scope default.');
```

i wonder if something along these lines would be more clear?